Formula/g/gromacs.rb

```diff
@@ -22,6 +22,7 @@ class Gromacs < Formula
   depends_on "cmake" => :build
   depends_on "fftw"
   depends_on "gcc" # for OpenMP
+  depends_on "muparser"
```

Comment:
When an existing muParser installation is used, `-DGMX_USE_MUPARSER=EXTERNAL` should be added to `args`, otherwise GROMACS will try to use its own version

Reply:
Our handling of external muParser turned out to be fragile, since we use different spelling for the target (fix attempted in https://gitlab.com/gromacs/gromacs/-/merge_requests/4959, but need more testing).

In the 2024.x branch, the internal muParser was used without any apparent issues, but was not the best choice for `brew` anyway (in part, because `gromacs` and `muparser` packages ended up conflicting when both were installed).